shared/pinentry/index.desktop.tsx

```diff
@@ -82,10 +82,9 @@ class Pinentry extends React.Component<Props, State> {
           ...globalStyles.flexBoxColumn,
           backgroundColor: globalColors.white,
           paddingBottom: globalMargins.medium,
-          paddingTop: globalMargins.medium,
```

Comment:
this is a random fix that fixes an issue @mlsteele found. i was already in here so...